### PR TITLE
feat: MongoDB vs DynamoDB vs Tigris blog post

### PR DESCRIPTION
**Pull request by leggetter**
- New blog post
- Updates newsletter subscription functionality

### CODE REVIEW

File: blog/2023-04-24-no-sql-database-comparison/index.mdx [hunk 1]
```diff
@@ -0,0 +1,347 @@
+---
+slug: mongodb-vs-dynamodb-vs-tigris-nosql-comparison
+title: MongoDB vs DynamoDB vs Tigris - a NoSQL Database Comparison
+description: >
```
Comment (leggetter) [inline blog/2023-04-24-no-sql-database-comparison/index.mdx:4]: Needs a meta description

File: blog/2023-04-24-no-sql-database-comparison/index.mdx [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+  </tr>
+</table>
+
+## Tigris: the MongoDB and DynamoDB alternative
+
+We built Tigris to give developers the consistency of ACID transactions with the
+flexibility of NoSQL at a fraction of the cost of alternatives. Compared to
+MongoDB and DynamoDB, Tigris gives you:
+
+- **Consistent data for mission-critical applications:** With Tigris, you can be
```
Comment (leggetter) [inline blog/2023-04-24-no-sql-database-comparison/index.mdx:324]: Are there more docs we can link to here? e.g. we have DB branching and MongoDB compatibility

File: blog/2023-04-24-no-sql-database-comparison/index.mdx [hunk 3]
```diff
@@ -0,0 +1,367 @@
...
+    </td>
+    <td>yes</td>
+    <td>no</td>
+  </tr>
+  <tr>
+    <td>
+      <strong>Self managed data</strong>
+    </td>
+    <td>yes</td>
+    <td>yes</td>
```
Comment (ovaistariq) [inline blog/2023-04-24-no-sql-database-comparison/index.mdx:325]: Wouldn't this be `no` for DynamoDB?

Comment (himank) [reply blog/2023-04-24-no-sql-database-comparison/index.mdx:325]: +1 this should be `no`.

Comment (leggetter) [reply blog/2023-04-24-no-sql-database-comparison/index.mdx:325]:
```suggestion
    <td>no</td>
```

File: blog/2023-04-24-no-sql-database-comparison/index.mdx [hunk 4]
```diff
@@ -0,0 +1,367 @@
...
+    </td>
+    <td>
+      <a href="https://www.tigrisdata.com/docs/concepts/architecture/">yes</a>
+    </td>
+    <td>yes</td>
+    <td>no</td>
+  </tr>
+  <tr>
+    <td>
+      <strong>Self managed data</strong>
```
Comment (leggetter) [inline blog/2023-04-24-no-sql-database-comparison/index.mdx:322]:
```suggestion
      <strong>Self-managed Data hosting</strong>
```

**Review by ovaistariq — APPROVED** [review]

**Pull request merged**